README.md

```diff
@@ -1 +1,46 @@
 # rancher-desktop-networking
+Is a simple layer 2 switch that allows connectivity between the VM and the host over `AF_VSOCK` written in Go using [gvisor's](https://github.com/google/gvisor) network stack and [gvisor-tap-vsock](https://github.com/google/gvisor). In addition to piping the ethernet frames, it also provides a configurable DNS, dynamic port forwarding and DHCP.
```

Comment:
> … In addition to piping the ethernet frames, it also provides a configurable DNS, dynamic port forwarding and DHCP.

That last part is asymmetrical, and _a… DNS_ doesn't seem to match. Consider something like:
>… In addition to piping the Ethernet frames, it also provides configurable DNS services, DHCP services, and dynamic port forwarding.

Reply:
It's not poetry it's documentation for a layer 2 switch 😂